py-polars/polars/io/parquet/functions.py

```diff
@@ -239,12 +241,22 @@ def scan_parquet(
     Examples
     --------
     >>> source = "s3://bucket/*.parquet"
+    >>> pl.scan_parquet(source)  # doctest: +SKIP
+    Pass in storage options to connect to the cloud provider:
     >>> storage_options = {
     ...     "aws_access_key_id": "<secret>",
     ...     "aws_secret_access_key": "<secret>",
     ...     "aws_region": "us-east-1",
     ... }
     >>> pl.scan_parquet(source, storage_options=storage_options)  # doctest: +SKIP
+    If you get a missing region error then set the region in the storage options:
```

Comment:
The region is already included in the previous example, so this does not add anything new.